src/net/download.rs

```diff
@@ -1696,65 +1746,58 @@ impl PeerNetwork {
     /// create the HTTP request.  Pops requests off the front of request_keys, and returns once it successfully
     /// sends out a request via the HTTP peer.  Returns the event ID in the http peer that's
     /// handling the request.
-    fn begin_request<F>(
+    pub fn begin_request<T: Requestable + std::fmt::Display>(
```

Comment:
Is there a way to remove the need for `std::fmt::Display` here?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/7a2d79b7b03d03615decb26eb4b639b1cf8ba345